REVIEWING.md

```diff
@@ -78,20 +78,13 @@ review, withdraws their hold, or is overruled by a preponderance of maintainers.
 Merging of PRs is done by the project maintainers.
 
 Like many open source projects, becoming a maintainer is based on contributions
-to the project. Active contributors to the project will eventually get noticed
-by the maintainers, and one of the existing maintainers will nominate the
-contributor to become a maintainer.  A 'yes' vote from >75% of
-the existing maintainers is required for approval.
-
-Removing a maintainer requires a 'yes' vote from >75% of the exsiting
-maintainers. Note that inactive maintainers might periodically be removed
-simply to keep the membership list accurate.
+to the project. Please see our [community roles](https://github.com/istio/community/blob/master/ROLES.md)
+document for information on how this is done.
```

Comment:
just realized that you don't have maintainer role in the community roles url.   you may want to change it to the mapping role.

Reply:
I'd like to wait on updating this doc until the current experiment with the CODEOWNERS stuff is done. If we fully switch to CODEOWNERS, then ROLES.md needs to be updated to merge the reviewer & approver roles, and the wording here will need to change too.

Reply:
SGTM.